static/app/views/replays/detail/timestampButton.tsx

```diff
@@ -6,27 +6,35 @@ import Tooltip from 'sentry/components/tooltip';
 import {IconPlay} from 'sentry/icons';
 import space from 'sentry/styles/space';
 
-type TimestampButtonProps = {
+type Props = {
+  format: 'mm:ss' | 'mm:ss.SSS';
   onClick: () => void;
   startTimestampMs: number;
   timestampMs: string | number | Date;
+  className?: string;
 };
 
-function TimestampButton({onClick, startTimestampMs, timestampMs}: TimestampButtonProps) {
+function TimestampButton({
+  className,
+  format,
```

Comment:
Should we give it a default format?

Reply:
I could do that

It was kind of a thought about how formatting _durations_ (not even timestamps) could be better. Typing strings in to `Date.format()` kinda sucks, but i don't mind so far having a list of supported/common formats there in the TS typeahead.